CHANGELOG.md

```diff
@@ -4,6 +4,302 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.16.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.16.0-rc1) - 2022-01-19
+
+* BREAKING
+  * Remove golang vendored directory (#18277)
+  * Paginate releases page & set default page size to 10 (#16857)
+  * Only allow webhook to send requests to allowed hosts (#17482)
+* SECURITY
+  * Sanitize user-input on file name (#17666)
+  * Use `hostmatcher` to replace `matchlist` to improve blocking of bad hosts in Webhooks (#17605)
+* FEATURES
+  * Add/update SMTP auth providers via cli (#18197)
+  * Support webauthn (#17957)
+  * Team permission allow different unit has different permission (#17811)
+  * Implement Well-Known URL for password change (#17777)
+  * Add support for ssh commit signing (#17743)
+  * Allow Loading of Diffs that are too large (#17739)
+  * Add copy button to markdown code blocks (#17638)
+  * Add .gitattribute assisted language detection to blame, diff and render (#17590)
+  * Add `PULL_LIMIT` and `PUSH_LIMIT` to cron.update_mirror task (#17568)
+  * Add Reindex buttons to repository settings page (#17494)
+  * Make SSL cipher suite configurable (#17440)
+  * Add groups scope/claim to OIDC/OAuth2 Provider (#17367)
+  * Add simple update checker to Gitea (#17212)
+  * Migrated Repository will show modifications when possible (#17191)
+  * Create pub/priv keypair for federation (#17071)
+  * Make LDAP be able to skip local 2FA (#16954)
+  * Add nodeinfo endpoint for federation purposes (#16953)
+  * Save and view issue/comment content history (#16909)
+  * Use git attributes to determine generated and vendored status for language stats and diffs (#16773)
+  * Add migrate from Codebase (#16768)
+  * Add migration from GitBucket (#16767)
+  * Add OAuth2 introspection endpoint (#16752)
+  * Add proxy settings and support for migration and webhook (#16704)
+  * Add microsoft oauth2 providers (#16544)
+  * Send registration email on user autoregistration (#16523)
+  * Defer Last Commit Info (#16467)
+  * Support unprotected file patterns (#16395)
+  * Add migrate from OneDev (#16356)
+  * Add option to update pull request by `rebase` (#16125)
+  * Add RSS/Atom feed support for user actions (#16002)
+  * Add support for corporate WeChat webhooks (#15910)
+  * Add a simple way to rename branch like gh (#15870)
+  * Add bundle download for repository (#14538)
+  * Add agit flow support in gitea (#14295)
+* API
+  * Add MirrorUpdated field to Repository API type (#18267)
+  * Fix NPE on try to get tag reference via API (#18245)
```

Comment:
This should be removed as it is regression and was not present in 1.15, so it's not needed in changelog

Reply:
```suggestion
```